add.go

```diff
@@ -112,6 +112,12 @@ func (b *Builder) Add(destination string, extract bool, options AddAndCopyOption
 	hostOwner := idtools.IDPair{UID: int(hostUID), GID: int(hostGID)}
 	dest := mountPoint
 	if destination != "" && filepath.IsAbs(destination) {
+		dir, _ := filepath.Split(destination)
```

Comment:
Why not use filepath.Dir(destination)?

Reply:
Great suggestion! Updated the code to use `Dir` instead.
Sorry for the delay :see_no_evil: